tests/acceptance/ENFormCreateCept.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+$I = new AcceptanceTester( $scenario );
+
+$I->wantTo( 'Check ENForm creation works' );
+
+$I->amOnPage( '/wp-admin/post-new.php?post_type=p4en_form' );
+
+$I->fillField('log', 'admin');
+$I->fillField('pwd', 'admin');
+$I->click('Log In');
+
+$I->see('Add New EN Form', 'h1');
+
+$I->fillField( 'post_title', 'Acceptance Test - ENForm' );
+
+$I->see( 'Form preview', 'span' );
+$I->see( 'Selected Components', 'span' );
+$I->see( 'Available Fields', 'span' );
+$I->see( 'Available Questions', 'span' );
+$I->see( 'Available Opt-ins', 'span' );
+
+$I->click(
+	[
+		'xpath' => '//button[@data-name="Email address"]',
```

Comment:
If this could be turned into css selectors easily would be nice.
Xpath selectors are the slowest